### DISCUSSION TITLE
v2/comment/zh/migration

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#382] [Announcements]
# v2/comment/zh/migration

从 @mr-hope/vuepress-plugin-comment 重命名为 vuepress-plugin-comment2。 ⚠ 新功能 添加 twikoo 和 giscus 评论服务的支持 !新增 (https://img.shields.io/badge/-New-brightgreen); 移除 Vssue 目前缺失 !警告 (https:...

https://vuepress-theme-hope.github.io/v2/comment/zh/migration.html

**Comment by maoba** [comment]
22
